pi-hole.json

```diff
@@ -0,0 +1,83 @@
+{
+    "Pi-Hole": {
+        "containers": {
+            "pi-hole-diginc": {
+                "image": "diginc/pi-hole",
+                "opts": [
+                    [
+                        "--cap-add",
+                        "NET_ADMIN"
+                    ],
+                    [
+                	"--restart",
+                	"always"
+            	    ]
+                ],
+                "launch_order": 1,
+                "ports": {
+                    "53": {
+                        "description": "DNS port. Required: 53",
+                        "host_default": 53,
+                        "label": "DNS-Port",
+                        "ui": false
+                    },
+                    "80": {
+                        "description": "Web-port. Recommended: 80",
+                        "host_default": 80,
+                        "label": "Web-Port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/etc/pihole": {
+                        "description": "Choose a Share for Plex Requests configuration. Eg: create a Share called pihole-config for this purpose alone.",
+                        "label": "Config Storage"
+                    }
+                },
+                "environment": {
+                    "ServerIP": {
+                        "description": "Enter IP-adress of rockstor server. If not specified it will default to internal docker IP and it will not work.",
+                        "label": "Rockstor IP",
+                        "index": 1
+                    },
+                    "VIRTUAL_HOST": {
+                        "description": "Enter IP of rockstor server.",
+                        "label": "Virtual Host",
+                        "index": 2
+                    },
+                    "DNS1": {
+                        "description": "Enter desired webpassword for pi-hole.",
+                        "label": "DNS Server 1",
+                        "index": 3,
+                        "default": "8.8.8.8"
+                    },
+                    "DNS2": {
+                        "description": "Enter desired webpassword for pi-hole.",
+                        "label": "DNS Server 2",
+                        "index": 4,
+                        "default": "8.8.4.4"
+                    },
+                    "IPv6": {
+                        "description": "Enter desired webpassword for pi-hole.",
+                        "label": "Enable IPv6",
+                        "index": 5,
+                        "default": "false"
```

Comment:
I have it as false on my setup and the web ui allows you to change this setting so probably best left default off.